6/ansible-6.build

```diff
@@ -30,29 +30,29 @@ community.ciscosmb: >=1.0.0,<2.0.0
 community.crypto: >=2.3.0,<3.0.0
 community.digitalocean: >=1.19.0,<2.0.0
 community.dns: >=2.1.0,<3.0.0
-community.docker: >=2.5.0,<3.0.0
+community.docker: >=2.6.0,<3.0.0
 community.fortios: >=1.0.0,<2.0.0
 community.general: >=5.0.0,<6.0.0
 community.google: >=1.0.0,<2.0.0
 community.grafana: >=1.4.0,<2.0.0
-community.hashi_vault: >=2.5.0,<3.0.0
+community.hashi_vault: >=3.0.0,<4.0.0
 community.hrobot: >=1.3.0,<2.0.0
 community.libvirt: >=1.1.0,<2.0.0
 community.mongodb: >=1.4.0,<2.0.0
 community.mysql: >=3.2.0,<4.0.0
 community.network: >=3.3.0,<4.0.0
```

Comment:
```suggestion
community.network: >=4.0.0,<5.0.0
```

community.network 4.0.0 has just been released. Should we add it? Otherwise, being a major release it would have to wait until ansible 7.